src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -86,6 +94,16 @@ func FindConfigFile(
 	}
 }
 
+// Returns the configured brokers list.
+// The configuration priority is as follows (highest to lowest):
+// 1. Values passed through flags (`brokers`)
+// 2. The addresses of a running local container cluster deployed with
```

Comment:
Isn't (2) actually `REDPANDA_BROKERS`, and then (3) is the container?

Reply:
You are quite right. I forgot to update this.